spec/controllers/restaurants_controller_spec.rb

```diff
@@ -0,0 +1,179 @@
+require 'rails_helper'
+
+RSpec.describe RestaurantsController, type: :controller do
+  def response_body
+    JSON.parse(response.body)
+  end
+
+  def create_cuisine
+    create(:cuisine, name: 'C1')
+  end
+
+  let(:restaurant1) {create(:restaurant, name: 'R1')}
```

Comment:
add 1 space after opening and before closing curly brackets. 
`{ create(:restaurant, name: 'R1') }`